bHY6yh-de5998/cod-bxxqqq_bHY6yh-de5998.xml

```diff
@@ -0,0 +1,127 @@
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 21</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 21</title>
+               <date when="2023-06-12">June 12, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+              <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-12" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="10-v"/>
+            <cb ed="#B" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de5998"><!-- p1l1-r21 -->
+            <head xml:id="bHY6yh-de5998-Hd1e108">Rubrica 21</head>
+            <head xml:id="bHY6yh-de5998-Hd1e111" type="question-title">De decimarum et primitiarum religione</head>
+            <p xml:id="bHY6yh-de5998-d1e114">
+              De decimarum 
+              <lb ed="#B"/>et primitiarum religione Rubrica 21
+    </p>
+            <p xml:id="bHY6yh-de5998-d1e121">
+              IN exodo capitulo 22<quote xml:id="bHY6yh-de5998-Qd1e125"> Decimas tuas et 
+              <lb ed="#B"/>primitias non tardabis offerre</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e133">
+              <lb ed="#B"/>/ <ref xml:id="bHY6yh-de5998-Rd1e130">23 capitulo</ref> <quote xml:id="bHY6yh-de5998-Qd1e137">Primitias frugum terre tue 
+              <lb ed="#B"/>deferre in dominum dei tui</quote>
+            </p>
+            <p xml:id="bHY6yh-de5998-d1e144"> /
```

Comment:
Can you double check these slashes. Often these get left over from the "venice transcription". If they should be deleted, please delete them. 

If the slashes really are in your manuscript, just let me know know and I will merge

Reply:
corrected